src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: bold;
+}
+
+:root {
+  --font-family: Avenir, Arial, sans-serif;
+  --font-color: #3d4e61;
+  --background-color: #fff;
+  --border-color: #e1e7ed;
+  --box-shadow: rgba(61, 78, 97, 0.1);
+  --hover-box-shadow: rgba(61, 78, 97, 0.2);
+  --placeholder-color: #3d4e61;
+  --search-icon-size: 19px;
+  --search-icon-small-size: 11px;
+  --search-bar-height: 70px;
+  --search-bar-small-height: 42px;
+  --font-size: 16px;
+  --font-size-small: 14px;
+}
+
+.page__body {
+  padding: 0;
+}
+
+.search-bar {
```

Comment:
Consider adding a bottom margin to the `.search-bar` instead of a top margin to be consistent with your margins, as per the checklist item 'Be consistent with your margins (Add only top or bottom, but not both)'.